WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -196,17 +198,72 @@ - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
 
+    if (self.splitViewControllerIsHorizontallyCompact) {
+        [self animateDeselectionInteractively];
+    }
+
     [self.headerView setBlog:self.blog];
 
     // Configure and reload table data when appearing to ensure pending comment count is updated
     [self configureTableViewData];
-    [self.tableView reloadData];
+
+    [self reloadTableViewPreservingSelection];
     [self preloadBlogData];
 }
 
+- (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection
+{
+    [super traitCollectionDidChange:previousTraitCollection];
+
+    // Required to update disclosure indicators depending on split view status
+    [self reloadTableViewPreservingSelection];
+}
+
+- (void)showDetailViewForSubsection:(BlogDetailsSubsection)section
+{
+    switch (section) {
+        case BlogDetailsSubsectionStats:
+            [self.tableView selectRowAtIndexPath:[NSIndexPath indexPathForRow:0 inSection:0]
```

Comment:
Maybe we go ahead and add a separate method for something like `indexPathForSubsection:(BlogDetailsSubsection)section`, just to keep things a bit cleaner and adaptable.